specs/commandline/policy.md

```diff
@@ -0,0 +1,671 @@
+
+```text
+Update the existing trust policies.
+
+Usage:
+  notation policy update [flags] <policy_name>
+
+Flags:
+      --cert-file          string   optional certificate filepath
+      --custom-level       string   optional custom level to based existing verification level, e.g. "authenticity=log,expiry=log"
+  -h, --help                        help for add
+      --id                 string   optional trust identity aka certificate subject info
+      --repo               string   optional repository URI
+      --trust-store        string   optional trust store in format "<trust_store_type>:<trust_store_name>", e.g. "ca:my_trust_store"
+      --verification-level string   optional verification level, options: strict, permissive, audit, skip
+```
+
+## Usage
+
+### Add a trust policy
+
+To add a trust policy, users need to specify the **policy name** as mandatory argument and configure other properties accordingly. The following table shows how to configure properties and default values.
+
+| Property Name       | Necessity | Default value | flag to configure               |
+| ------------------- | --------- | ------------- | ------------------------------- |
+| `trustStores`       | Required  | N/A           | `--trust-store`                 |
+| `registryScopes`    | Optional  | `"*"`         | `--repo`                        |
+| `level`             | Optional  | `"strict"`    | `--verification-level`          |
+| `override`          | Optional  | nil           | `--custom-level`                |
+| `trustedIdentities` | Optional  | `"*"`         | `--id` or `--cert-file` or both |
+
+```shell  
+notation policy add --repo "dev.wabbitnetworks.io/net-monitor" --trust-store "ca:wabbit-network-dev" --id "C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io" --verification-level "strict" wabbit-network-dev
+```
+
+The execution of `add` fails in one of below cases:
+
+- The policy name exists.
+- There can only be one trust policy that uses a global scope, that is, the value of `registryScopes` is `*`.
+
+Upon successful execution, the added trust policy is printed out. For example:
+
+In text format
+
+```text
+"name": "wabbit-networks-dev"
+"registryScopes": "dev.wabbitnetworks.io/net-monitor"
+"level": "strict"
+"trustStores": "ca:wabbit-networks-dev"
+"trustedIdentities": "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io"
+```
+
+In json format
+
+```json
+{
+  "name": "wabbit-networks-dev",                              
+  "registryScopes": [ "dev.wabbitnetworks.io/net-monitor" ],                                  
+  "signatureVerification": {                                  
+      "level": "strict"
+  },
+  "trustStores": [ "ca:wabbit-networks-dev" ],                
+  "trustedIdentities": [                                      
+      "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io"
+  ]
+}
+```
+
+### Add a trust policy with default values
```

Comment:
It seems that we are missing the scenario `notation policy add --policy-file policy_file_path`?